.github/workflows/js_api.yml

```diff
@@ -0,0 +1,101 @@
+name: JS API
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'src/bindings/js/**'
+      - '.github/workflows/js_api.yml'
+  push:
+    paths:
+      - 'src/bindings/js/**'
+      - '.github/workflows/js_api.yml'
+    branches:
+      - master
+      - 'releases/**'
+
+concurrency:
+  # github.ref is not unique in post-commit
+  group: ${{ github.event_name == 'push' && github.run_id || github.ref }}-js-api
+  cancel-in-progress: true
+
+jobs:
+  JS_API:
+    name: Configure and Test JS API Bindings
+    strategy:
+      max-parallel: 3
+      fail-fast: false
+      matrix:
+        include:
+          - machine: 'ubuntu-22.04'
+            shell: 'bash'
+# TODO: uncomment once they work
+#          - machine: 'windows-2022'
+#            shell: 'cmd'
+#          - machine: 'macos-11'
+#            shell: 'bash'
+    defaults:
+      run:
+        shell: ${{ matrix.shell }}
+    runs-on: ${{ matrix.machine }}
+    env:
+      OPENVINO_REPO: "${{ github.workspace }}/openvino"
+      JS_BINDINGS_DIR: "${{ github.workspace }}/openvino/src/bindings/js"
+    steps:
+      - name: Clone OpenVINO
+        uses: actions/checkout@v4
+        with:
+          path: 'openvino'
+
+      #
+      # Init submodules
+      #
+
+      - name: Install openvino dependencies
+        if: ${{ contains(matrix.machine, 'ubuntu-22.04') }}
+        working-directory: ${{ env.OPENVINO_REPO }}
+        run: git submodule update --init --recursive
+
+      #
+      # Install openvino dependencies
+      #
+
+      - name: Install openvino dependencies
+        if: ${{ contains(matrix.machine, 'ubuntu-22.04') }}
+        working-directory: ${{ env.OPENVINO_REPO }}
+        run: sudo -E apt update && sudo ./install_build_dependencies.sh
+
+      #
+      # Build openvino and install js bindings
+      #
+
+      - name: Configure openvino build
+        if: ${{ contains(matrix.machine, 'ubuntu-22.04') }}
+        working-directory: ${{ env.OPENVINO_REPO }}
+        run: mkdir build && cd build && cmake -DCPACK_GENERATOR=NPM -DENABLE_SYSTEM_TBB=OFF -UTBB* -DCMAKE_INSTALL_PREFIX=../src/bindings/js/node/bin ..
```

Comment:
I suppose we don't need to build OpenVINO specifically for node JS and can integrate into existing workflows for Windows, Linix